js/plugin/services.ts

```diff
@@ -89,22 +100,30 @@ export const ServicesMixin = (SuperClass) => {
 
         case "notification":
           {
-            const { message, action_text, action, duration, dismissable } =
+            data.action_action = data.action;
+            delete data.action;
+            var { message, action_text, action_action, duration, dismissable } =
```

Comment:
Same here. Why not `const`?

Reply:
`action_action` is assigned if a string and not list/array.

Reply:
I see now. Thanks.